extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authentication/db/cache/CoordinatorPollingBasicAuthenticatorCacheManager.java

```diff
@@ -113,26 +116,51 @@ public void start()
           () -> {
             try {
               long randomDelay = ThreadLocalRandom.current().nextLong(0, commonCacheConfig.getMaxRandomDelay());
-              LOG.debug("Inserting random polling delay of [%s] ms", randomDelay);
+              LOG.debug("Inserting cachedUserMaps random polling delay of [%s] ms", randomDelay);
               Thread.sleep(randomDelay);
 
-              LOG.debug("Scheduled cache poll is running");
+              LOG.debug("Scheduled user cache poll is running");
               for (String authenticatorPrefix : authenticatorPrefixes) {
                 Map<String, BasicAuthenticatorUser> userMap = fetchUserMapFromCoordinator(authenticatorPrefix, false);
                 if (userMap != null) {
                   cachedUserMaps.put(authenticatorPrefix, userMap);
                 }
               }
-              LOG.debug("Scheduled cache poll is done");
+              LOG.debug("Scheduled user cache poll is done");
             }
             catch (Throwable t) {
               LOG.makeAlert(t, "Error occured while polling for cachedUserMaps.").emit();
             }
           }
       );
 
+      ScheduledExecutors.scheduleWithFixedDelay(
```

Comment:
nit: probably its ok to poll both config and userMap in same scheduler.

Reply:
this is now irrelevant since we aren't polling for configs anymore, code has be removed